.github/workflows/ci.yml

```
@@ -20,6 +20,8 @@ jobs:
           - '3.0'
           - '3.1'
           - '3.2'
+          - '3.3'
+          - '3.4'
           - 'head'
```

```suggestion
          - 'head'
```

I approved the test run. Looks like `head` needs to be removed.